code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -423,6 +423,12 @@
 
 /datum/reagent/medicine/liquid_solder/on_mob_life(mob/living/M)
 	M.adjustBrainLoss(-3*REM)
+	if(iscarbon(M))
```

Comment:
Won't this allow all carbons to use it? Or is it supposed to be like that

Reply:
it checks for the reagent processing tag first.

Reply:
But, after that, there aren't any IPC checks as far as I can see.

Reply:
if the person isnt a synthetic, it wont process the reagent at all.

Reply:
Oh, didn't see the `process_flags = SYNTHETIC` above. Apologies

Reply:
its okay bro.